airflow/executors/kubernetes_executor.py

```diff
@@ -530,13 +532,15 @@ def start(self) -> None:
             self.kube_config.worker_pods_pending_timeout_check_interval,
             self._check_worker_pods_pending_timeout,
         )
-        self.event_scheduler.call_regular_interval(
-            self.kube_config.worker_pods_queued_check_interval,
-            self.clear_not_launched_queued_tasks,
-        )
-        # We also call this at startup as that's the most likely time to see
-        # stuck queued tasks
-        self.clear_not_launched_queued_tasks()
+
+        if self.job_id != 'manual':
```

Comment:
Any reason why this excludes `manual`?

Reply:
@snjypl personally, I'll prefer to have this part in a separate PR because it seems to be addressing something else?